pynest/nest/tests/test_stdp_nn_synapses.py

```diff
@@ -195,7 +195,7 @@ def reproduce_weight_drift(self, _pre_spikes, _post_spikes,
                     # with t_previous_pre, thus due to the restricted
                     # pairing scheme we do not account it.
                 if (self.synapse_parameters["synapse_model"] == "stdp_nn_symm_synapse" or
-                        self.synapse_parameters["synapse_model"] == "stdp_nn_pre-centered_synapse"):
+                        self.synapse_parameters["synapse_model"] == "stdp_nn_pre_centered_synapse"):
```

Comment:
Do you think we need to document this change in synapse name in the NEST 2x -> NEST 3 documentation? It is very minor, so I do not know if it is necessary.

Reply:
Yes, that would be great to mention, cheers.